skiko/src/awtMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.kt

```diff
@@ -74,7 +79,9 @@ internal class MetalRedrawer(
             setVSyncEnabled(device, enabled = false)
             update(System.nanoTime())
             performDraw()
```

Comment:
Shouldn't we check it here? `update` calls `render`, not `performDraw`

Reply:
Not sure that I got what you mean. `perfromDraw` also contains the `isDisposed` check inside. So the only problem is  `setVSyncEnabled`

Reply:
But how it possible to layer be disposed? `performDraw` don't call user code.

If it is still disposed in `update`, then for safety reasons it is better to put check above, as reading the current code we can think that we added a check for a case when we dispose in `performDraw`, and people can miss this check when they add new code before `performDraw`

Reply:
Ok, right, let's move it above